draft-trammell-taps-interface.md

```diff
@@ -504,7 +517,9 @@ interface configured in the system policy.
 
 ### Capacity Profile {#prop-cap-profile}
 
-This enumerated property specifies the application's expectation of the
+Type: Enumeration 
```

Comment:
As above.

Reply:
> The application tells the Transport system the traffic is Constant Rate, not that it prefers / wants to avoid that the traffic is treated as constant rate.

How is the TAPS system's action based on this information different, though?

Reply:
This case is different from the Interface Type case:
The Interface type case needs a parameter in addition to the preference.
For the Capacity Profile, the preference makes no difference here, adding it is redundant and mildly confusing. So I prefer it being just a Enum of the envisioned capaity profiles.

Reply:
okay, that makes sense. There should be some language, either here or in Implementation, that a selection of a capacity profile may be used by the TAPS system as a "preference" for a transport that is "good" for the given capacity profile.